warehouse/query-core/src/main/java/datawave/query/tables/ShardQueryLogic.java

```diff
@@ -2049,7 +2143,11 @@ public Set<String> getOptionalQueryParameters() {
         optionalParams.add(datawave.webservice.query.QueryParameters.QUERY_PAGESIZE);
         optionalParams.add(datawave.webservice.query.QueryParameters.QUERY_PAGETIMEOUT);
         optionalParams.add(datawave.webservice.query.QueryParameters.QUERY_EXPIRATION);
-        optionalParams.add(datawave.webservice.query.QueryParameters.QUERY_MAX_RESULTS_OVERRIDE);
```

Comment:
Was this removal intentional?

Reply:
It was not, I think this got missed when resolving some conflicts with updates from integration. I'll put it back in.